src/engine/test/source/hlp/hlp_test.cpp

```diff
@@ -636,3 +645,82 @@ TEST(filepath_test, force_unix_format)
     ASSERT_EQ("test\\1:2.txt", result["_file.name"]);
     ASSERT_EQ("txt", result["_file.extension"]);
 }
+
+TEST(userAgentTests, user_agent_firefox)
```

Comment:
follow the new naming convention of the tests, this should be renamed as 
"(hlpTests_UserAgent, firefox_user_agent)"
Same for below tests.